simulation_parameters/microbe_stage/membranes.json

```diff
@@ -28,6 +30,7 @@
         "normalTexture": "res://assets/textures/CellWallGradient.png",
         "damagedTexture": "res://assets/textures/CellWallGradientDamaged.png",
         "movementFactor": 0.8,
+        "movementWigglyness": 0.8,
```

Comment:
I think this should be set to the same value as chitin, as the cell wall is not meant to be so wiggly